### PR TITLE
Additional EP600 values

### PR DESCRIPTION
**Pull request by KM011092**
EP600 additions:
- Consumption values
- Grid, Consumptions and Solar Totals (as displayed on the first page of the Bluetti APP)

### CODE REVIEW

**Comment by Patrick762** [comment]
Could you remove the IntelliJ directory?

**Comment by KM011092** [comment]
> Could you remove the IntelliJ directory?
Not sure if I understand what you want me to remove. I am not aware that I‘ve added an IntelliJ directory. Can you please be a little more specific about what you want me to remove ?

**Comment by Patrick762** [comment]
I mean the .idea directory

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (Patrick762) [inline .idea/.gitignore:1]: Please remove this file

File: .idea/hassio-bluetti-bt-KM1.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Patrick762) [inline .idea/hassio-bluetti-bt-KM1.iml:1]: Please remove this file

File: .idea/inspectionProfiles/profiles_settings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (Patrick762) [inline .idea/inspectionProfiles/profiles_settings.xml:1]: Please remove this file

File: .idea/misc.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Patrick762) [inline .idea/misc.xml:1]: Please remove this file

File: .idea/modules.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Patrick762) [inline .idea/modules.xml:1]: Please remove this file

File: .idea/vcs.xml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Patrick762) [inline .idea/vcs.xml:1]: Please remove this file

**Review by Patrick762 — CHANGES_REQUESTED** [review]

**Review by Patrick762 — APPROVED** [review]

**Pull request merged**